deepomatic/http_helper.py

```diff
@@ -37,7 +37,7 @@
 
 
 class HTTPHelper(object):
-    def __init__(self, app_id, api_key, verify, host, version, check_query_parameters, user_agent_suffix=''):
+    def __init__(self, app_id, api_key, verify, host, version, check_query_parameters, user_agent_suffix='', pool_maxsize=100):
```

Comment:
isn't 100 too much? maybe should be customizable by the user

Reply:
I will put 50 then, it's just a theoric maximum. I will add it to the client constructor as well.